### PR TITLE
docs(understanding-deis): document database lifecycle

### PR DESCRIPTION
**Pull request by bacongobbler**
closes deis/postgres#34

### CODE REVIEW

**Comment by slack** [comment]
The backup and restore doc is still TODO in master.

**Comment by bacongobbler** [comment]
I don't know enough about minio to comment on how minio should be properly backed up to persistent disk, so I left that out in this PR.

File: docs/src/understanding-deis/components.md [hunk 1]
```diff
@@ -41,18 +42,22 @@ The router component uses [Nginx][] to route traffic to application containers.
 
 ## Store
 
-The store component uses [Ceph][] to store data for Deis components
-which need to store state, including [Registry][], [Database][].
+The Store component uses [Minio][] to store data for Deis components
```
Comment (arschles) [inline docs/src/understanding-deis/components.md]: this should probably say that the store component uses minio by default - you can configure the cluster to use other object stores

**Comment by slack** [comment]
Object storage should generally be transparent to the process of restoring a postgres backup, no?

File: docs/src/understanding-deis/components.md [hunk 2]
```diff
@@ -41,18 +42,22 @@ The router component uses [Nginx][] to route traffic to application containers.
...
+which need to store state, including the [Registry][] and the [Database][].
+
+It is highly recommended that the Store's data is backed by a persistent volume in the event of
```
Comment (arschles) [inline docs/src/understanding-deis/components.md]: same idea here - if you _do_ use minio, you should back it by a persistent volume

File: docs/src/understanding-deis/components.md (hunk 1 above)
Comment (bacongobbler) [inline docs/src/understanding-deis/components.md]: good point

**Comment by bacongobbler** [comment]
@slack yes, generally it is completely transparent. The user doesn't need to know anything about the underlying object storage layer we provide unless they swap it out with S3.

**Pull request merged**